source/index.md

```diff
@@ -372,18 +380,30 @@ Example response (snippet):
        "code": 20000
    },
    "processed_at": "2014-07-09T13:33:44+00:00",
-   "recurring": true
+   "recurring": true,
+   "_embedded": { "card": { "id": "58dabba0-e9ea-4133-8c38-bfa1028c1ed2" } }
 }
 ````
 
-Above can be repeated whenever you need to reserve money and must be followed
-by a capture transaction.
+This should be followed by a capture except when the amount is `0`.
+
+Subsequent authorizations are made similarly, but obviously neither CSC nor
```

Comment:
We should flip the order of **Recurring payments** and **3-D Secure** since **Recurring payments** reference `pares`.
A note "_See 3-D Secure for information on `pares`_" would do as well.


But writing "_obviously_ neither CSC nor PARes would be included" is a bit on the nose, since the reader have not read the **3-D Secure section** yet, and familiarity  of `PARes` is not as widescread as `PAN` or `CSC`, so including/excluding `PARes` is not obvious. :)